src/components/pager/js/Pager.js

```diff
@@ -295,8 +295,12 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     });
 
     fluid.pager.previousNext.update = function (that, disabledStyle, newModel) {
-        that.previous.toggleClass(disabledStyle, newModel.pageIndex === 0);
-        that.next.toggleClass(disabledStyle, newModel.pageIndex === newModel.pageCount - 1);
+        var previousToggle = (newModel.pageIndex === 0);
+        var nextToggle = (newModel.pageIndex === newModel.pageCount - 1);
```

Comment:
I do still think you need to rename "previousToggle" and "nextToggle" as they do not refer to the toggles but are related to their states.